src/DataViews/include/DataViews/DataView.h

```diff
@@ -104,6 +104,35 @@ class DataView {
   [[nodiscard]] DataViewType GetType() const { return type_; }
   [[nodiscard]] virtual bool ResetOnRefresh() const { return true; }
 
+  // Hooking related actions
+  static const std::string kMenuActionLoadSymbols;
```

Comment:
Would it work to make these `constexpr std::string_view`s and completely define them in the header? These `const std::string` always felt wrong to me.

Reply:
Sorry for bringing this up so late. :see_no_evil: